site/src/comparison.rs

```diff
@@ -486,21 +490,30 @@ pub fn write_summary_table(
         render_stat(secondary.num_improvements, || Some(
             secondary.arithmetic_mean_of_improvements()
         )),
-        primary.arithmetic_mean_of_changes()
+        if primary.is_empty() {
+            "N/A".to_string()
+        } else {
+            format!("{:.1}%", primary.arithmetic_mean_of_changes())
+        }
     )
     .unwrap();
 
-    let largest_change = primary
-        .most_relevant_changes()
-        .iter()
-        .fold(0.0, |accum: f64, item| {
-            let change = item.map(|v| v.relative_change() * 100.0).unwrap_or(0.0);
-            accum.max(change)
-        });
+    let largest_change = if primary.is_empty() {
+        "N/A".to_string()
+    } else {
+        let change = primary
+            .most_relevant_changes()
+            .iter()
+            .fold(0.0, |accum: f64, item| {
```

Comment:
You should be able to use [`sum`](https://doc.rust-lang.org/std/iter/trait.Sum.html#tymethod.sum) here instead of `fold`.

Reply:
I assume that you meant `max`? Good point, I'll change it.

Reply:
Ah, actually `max` can't be used because `f64` doesn't implement `Ord`. That's also why I originally used `fold` :D

Reply:
Fair enough.